timeout_decorator/timeout_decorator.py

```diff
@@ -65,6 +62,10 @@ def new_function(*args, **kwargs):
                 if new_seconds:
                     old = signal.signal(signal.SIGALRM, handler)
                     signal.setitimer(signal.ITIMER_REAL, new_seconds)
+
+                if not seconds:
```

Comment:
otherwhise we return the original function if the seconds parameter on the Timeout is none at the time we create the decorator.